blaze/compute/tests/test_csv_compute.py

```diff
@@ -37,25 +37,23 @@ def test_compute_chunks_on_single_csv():
     csv = CSV(example('iris.csv'))
     s = symbol('s', discover(csv))
     expr = s.sepal_length.max()
-    assert compute(expr, {s: csv}, comfortable_memory=10, chunksize=50) == 7.9
+    assert compute(expr, {s: csv}, comfortable_memory=10, blocksize=50) == 7.9
 
 
-def test_pre_compute_with_projection_projects_on_data_frames():
+def test_compute_with_projection_projects_on_data_frames():
     csv = CSV(example('iris.csv'))
     s = symbol('s', discover(csv))
-    result = pre_compute(s[['sepal_length', 'sepal_width']].distinct(),
-                         csv, comfortable_memory=10)
-    assert set(first(result).columns) == \
-            set(['sepal_length', 'sepal_width'])
+    result = compute(s[['sepal_length', 'sepal_width']],
+                     csv, comfortable_memory=10)
+    assert set(result.columns) == set(['sepal_length', 'sepal_width'])
 
 
-def test_pre_compute_calls_lean_projection():
+def test_compute_calls_lean_projection():
     csv = CSV(example('iris.csv'))
     s = symbol('s', discover(csv))
-    result = pre_compute(s.sort('sepal_length').species,
-                         csv, comfortable_memory=10)
-    assert set(first(result).columns) == \
-            set(['sepal_length', 'species'])
+    result = compute(s[s.sepal_length > 5.0].species,
+                     csv, comfortable_memory=10)
+    assert len(result) == 118
```

Comment:
To verify we're doing the lean projection, this needs to be something like:

```python
result = pre_compute(s[s.sepal_length > 5.0].species, csv, comfortable_memory=10)
assert set(result.columns) == set(['sepal_length', 'species'])
```

Specifically, we need to test the `pre_compute()` stage explicitly.